src/components/UI/Input/Input.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styleClasses from './Input.module.css';
+
+const Input = React.forwardRef((props, ref) => {
```

Comment:
you can import everything on the top line like:
`import React { useRef, forwardRef } ` from 'react';